frontend/public/style.scss

```diff
@@ -10,7 +10,8 @@
 @import "~bootstrap-sass/assets/stylesheets/bootstrap/variables";
 @import "~bootstrap-sass/assets/stylesheets/bootstrap/mixins";
 @import "~font-awesome-sass/assets/stylesheets/font-awesome/variables";
-@import "~xterm/dist/addons/fullscreen/fullscreen.css";
+@import "~xterm/dist/addons/fullscreen/fullscreen";
```

Comment:
Looks like this leaked from your other PR :)

Reply:
Actually... I realize now that merged. Did we have a bug in how we imported this? What effect does fixing that have?

Let's fix that as a separate PR. That should go in 3.11.

Reply:
> Did we have a bug in how we imported this? What effect does fixing that have?

Is there something wrong with how it is imported?

Reply:
Well you changed the line here and it doesn't look related to service classes. I'm trying to understand why the change was made.

Reply:
OK, I'd rather make this a separate PR against master so it's fixed in 3.11 too

Reply:
Ok, reverted.